roles/iptablesrules/templates/team.iptables.rules.j2

```diff
@@ -60,15 +60,19 @@
 {% if enable_reverseproxy -%}
 #TODO: make the ip for reverseproxy a lookup/variable/something better than hardcoded
 # http(s) to/from reverseproxy
--A INPUT -p tcp --sport 80 -s 10.3.3.205/32 -j ACCEPT
--A OUTPUT -p tcp --dport 80 -d 10.3.3.205/32 -j ACCEPT
--A INPUT -p tcp --sport 443 -s 10.3.3.205/32 -j ACCEPT
--A OUTPUT -p tcp --dport 443 -d 10.3.3.205/32 -j ACCEPT
+-A INPUT -p tcp --sport {{ ccs_port }} -s {{ ccs_ip }}/32 -j ACCEPT
+-A OUTPUT -p tcp --dport {{ ccs_port }} -d {{ ccs_ip }}/32 -j ACCEPT
```

Comment:
it seems these should the reverseproxy ip not the ccs_ip ?

Reply:
yeah, I think you're correct. sill fix in a moment.

Reply:
fixed.